openreview/conference/builder.py

```diff
@@ -182,6 +188,12 @@ def set_homepage_layout(self, layout):
     def set_double_blind(self, double_blind):
         self.double_blind = double_blind
 
+    def set_subject_areas(self, subject_areas):
+        self.subject_areas = subject_areas
+
+    def get_subject_areas(self):
```

Comment:
my opinion is that we don't need getters and setters like this, unless we want to do some kind of validation -- we can just set the subject_areas attribute directly

Reply:
I'm not sure what I'm going to do with getters and setters. In some cases I need them, in other cases I don't. I want to keep the same line.